tools/Statistics/CGAL/CGALTypes.h

```diff
@@ -0,0 +1,26 @@
+#ifndef PYMESH_CGALTYPES_H
+#define PYMESH_CGALTYPES_H
+
+#include <CGAL/Exact_predicates_exact_constructions_kernel.h>
+#include <CGAL/array.h>
+#include <CGAL/Polyhedron_3.h>
+#include <CGAL/Surface_mesh.h>
+#include <CGAL/Simple_cartesian.h>
+#include <CGAL/Nef_polyhedron_2.h>
+#include <CGAL/Point_3.h>
+
+typedef std::array<double, 3>                           Point_type;
+typedef std::array<int, 3>                              Polygon_type;
+typedef CGAL::Simple_cartesian<double>                  Kernel;
```

Comment:
This kernel is not safe if you call functions like `does_self_intersect()` as exact predicates are needed to return a certified result. `CGAL::Exact_predicates_inexact_construction_kernel` is a better choice.

Reply:
Good to know, I'll change this.